AwesomeProject/screens/Menu/Menu.screen.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import {NavigationContainer} from '@react-navigation/native';
+import {Button, View, ScrollView} from 'react-native';
+import styles from '../Menu/Menu.styles';
+import AboutMe from '../AboutMe/AboutMe.screen';
+
+export default Screens = () => (
+  <NavigationContainer>
+    <Stack.Navigator initialRouteName="Screens">
+      <Stack.Screen name="Menu" component={MenuScreen} />
```

Comment:
create 'config' folder on the same level as 'screens' and create constants.js file.
MENU_SCREEN and ABOUT_ME_SCREEN and import them here and pass to name property.